pkg/controller/plan/kubevirt.go

```diff
@@ -1336,23 +1339,33 @@ func (r *KubeVirt) podVolumeMounts(vmVolumes []cnv.Volume, configMap *core.Confi
 
 	switch r.Source.Provider.Type() {
 	case api.Ova:
-		server := r.Source.Provider.Spec.URL
-		splitted := strings.Split(server, ":")
-
-		if len(splitted) != 2 {
-			r.Log.Info("The NFS server path format is wrong")
+		pvcName := fmt.Sprintf("ova-nfs-pvc-%s-%s", r.Source.Provider.Name, r.Plan.Name)
+		/*
+			TODO: set ownerReferences the PV and PVC deleted once the plan is done.
+			cross namespaces are not possible to be used for plan/provider ownership.
+			optional: PV+PVC per VM, deleted via cleanup. ownership: importer pod? DV?
+			ownerReference := meta.OwnerReference{
```

Comment:
@liranr23 and I talked about this part earlier today - I'd check if it's possible to set the provider as the owner of a PV that is shared by the server and conversion pods, and remove PVCs during plan's cleanup

after a second thought, what if we migrate to a remote cluster? in that case, we need to create the PV for the conversion pod..

Reply:
I've tested this scenario before my PTO (to save some PV creation stages) but once the PV is bound to a PVC its not possible to use it for another one, even if it set as read-write-many.
so I think the best will be to keep the creation + clean up as now or change the PV delete policy so it will be deleted once we deleting the PVC, and additionally add labels of the plan and the providers for the PVC + PV.

Reply:
in that case, i think we should create 1 PV and 1 PVC per plan on destination with labels of the plan. i think it's enough to remove them when archiving.

Reply:
> I've tested this scenario before my PTO (to save some PV creation stages) but once the PV is bound to a PVC its not possible to use it for another one, even if it set as read-write-many.

this seems suspicious - do you still have this environment?

Reply:
> in that case, i think we should create 1 PV and 1 PVC per plan on destination with labels of the plan. i think it's enough to remove them when archiving.

right, but the label should refer to the migration

Reply:
@ahadas its not supported, I did double check and search it up, the shared property referring to the fact that many pods can use the same PVC and PV and not that more then one PVC can use a PV.
![Screenshot from 2023-10-02 16-16-39](https://github.com/kubev2v/forklift/assets/73168733/ff2e7faa-e977-4fae-9143-50f0cbc19c7a)

Reply:
alright, if we no longer try to reuse the PV and don't set owner-references then it's fine